apps/studio/prisma/seed.ts

```diff
@@ -146,7 +146,28 @@ async function main() {
   const { id: siteId } = await db
     .insertInto("Site")
     .values({
-      name: "Ministry of Trade and Industry",
+      name: "Isomer",
+      theme: jsonb({
+        colors: {
+          brand: {
+            canvas: {
+              alt: "#bfcfd7",
+              default: "#e6ecef",
+              inverse: "#00405f",
+              backdrop: "#80a0af",
+            },
+            interaction: {
+              hover: "#002e44",
+              default: "#00405f",
+              pressed: "#00283b",
+            },
+          },
+          "site-primary-100": "#fef4e8",
+          "site-primary-200": "#ffeec2",
+          "site-primary-default": "#f78f1e",
+          "site-secondary-default": "#877664",
```

Comment:
nit: I think these are no longer needed now since we have shifted all our components over to the colour tokens

Reply:
removed